balancer_switching_test.go

```diff
@@ -28,6 +28,7 @@ import (
 	"google.golang.org/grpc/balancer"
 	"google.golang.org/grpc/balancer/roundrobin"
 	"google.golang.org/grpc/internal"
+	"google.golang.org/grpc/internal/balancer/stub"
```

Comment:
For generically-named things ("stub"), maybe we should try to make sure the package name (not including path) is more descriptive?  E.g. "stubbalancer".  WDYT?  This is similar to "xds/client" vs. "xds/xdsclient".  There's some stutter in the path, but I think it's more important to have good package names, like "ioutil".

(Not an actionable comment for this PR)

Reply:
Does the go style discourage stutter in path? Maybe we should just make the package names looooong

Avoiding stutter in types/functions makes sense, because they are always used with the package (`xdsclient.New`, so no need for `xdsclient.NewXDSClient`).

But package names are never used with the path, so the name itself has to be informative.

Reply:
Yes, the only portion of the path that is used in call sites is the last part. So, having stutter in the path should be fine where it adds clarity at the usage sites.

I don't think there is specific guidance on this on the Go style. The only guidance is to avoid stutter with the package name and its exported types.